INSTALL.md

```diff
@@ -309,8 +309,25 @@ Run [Yarn](https://www.yarnpkg.com/) to install all the required dependencies:
 yarn install
 ```
 
+## Manual setup for Arch linux
+
+The setup is the same as for Fedora, but the naming of packages might differ.
+Since Porta needs ImageMagick6, you need to install [libmagick6](https://aur.archlinux.org/packages/libmagick6/) from AUR.
+
+When running `bundle install`, you need to specify `PKG_CONFIG_PATH`.
+```shell
+PKG_CONFIG_PATH=/usr/lib/imagemagick6/pkgconfig bundle install
+```
+
 ## Setup Database
 
+If you are using a different database than MySQL you need to export the env variable `DATABASE_URL`.
```

Comment:
Well not really, basically this is configuration driven, you pick the configuration you need and that's it

The `config/examples/*.yml` files are, as named, _examples_
They are default values that can run the application by default and also indicators of what could be done/modified

I would not state this as a step to run postgresql because you can just modify the `config/database.yml` file at your need

Reply:
So we should say that in order to run porta with a different DB they need to change `config/database.yml`.
Because the DB is chosen by the `DATABASE_URL` value.

The problem for me was that in the steps for fedora there is postgres as the database and there was no step that pointed that I should change something in order to run the porta with postgres.

Reply:
Well it is opinionated, there is no "1 way to do it". You pick the one you want.
If you wish you could describe it more generically for the 3 databases but ... it is how Rails does it so that is why there is not really need of mentioning that.

I really mean that if somebody wants to develop on this project, they MUST know Rails config/database.yml

I'd prefer putting in INSTALL.md what we really support and step to install on few OS, and create a Troubleshooting wiki page

Reply:
I agree that INSTALL.md should contain only what we really support. I will move the specific stuff to the wiki page.
However, the installation guide for Fedora shows steps to deploy Postgres and never mentions that you need to modify `/config/database.yml`.
I would propose changing Postgres to MySQL in the INSTALL.md and creating a wiki page for the setup of different DBs.